adafruit_requests.py

```diff
@@ -399,10 +447,31 @@ def _send(socket: SocketType, data: bytes):
                 # Not EAGAIN; that was already handled.
                 raise OSError(errno.EIO)
             total_sent += sent
+        return total_sent
```

Comment:
this was missing

Reply:
Is it related to the rest of these changes or just something you noticed while working on this?

Is there some reference somewhere that documents it's need to return here? I only had a quick look but I didn't see any spots where `self._send()` is called that do anything with a return value from it so I'm guessing it won't make any actual difference as it is used now?

Reply:
Is there a reference to somewhere that shows it should be there? like a link to the same thing inside of cpython requests, or any documentation somewhere? I'm not really doubting you, but I don't have any real insight into it so I have no way to really know whether it should or shouldn't. 

In any case if it is added, the type annotations should be updated to show the return type ideally I think.